actionsystemtest/lib/action_system_test/test_helpers/screenshot_helper.rb

```diff
@@ -0,0 +1,56 @@
+module ActionSystemTest
+  module TestHelpers
+    # Screenshot helper for system testing
+    module ScreenshotHelper
```

Comment:
I like it. I think it's exactly the kind of process improvement that makes the out-of-the-box configuration feels extra nice. We use those screenshots on every failure for BC3 to help diagnose.